root_locals.tf

```diff
@@ -105,4 +105,6 @@ locals {
   tmp_directory         = "/tmp"
   consignment_user_name = "consignment_api_user"
 
+  //feature access blocks
+  block_http4s = local.environment == "prod" || local.environment == "staging" ? true : false
```

Comment:
Could rewrite this to: 

`block_http4s = local.environment == "intg" ? false : true`

But how it is will be fine, as this is temporary code anyway.